taipy/config/common/scope.py

```diff
@@ -43,33 +43,36 @@ class Scope(_OrderedEnum):
     - `CYCLE`
     - `SCENARIO` (Default value)
 
-    Each data node config has a scope. It is an attribute propagated to the `DataNode^` when instantiated from
-    a `DataNodeConfig^`. The scope is used to determine the _visibility_ of the data node, and which scenarios can
-    access it.
+    Each data node config has a scope. It is an attribute propagated to the `DataNode^`
+    when instantiated from a `DataNodeConfig^`. The scope is used to determine the
+    _visibility_ of the data node, and which scenarios can access it.
```

Comment:
```suggestion
    _accessibility_ of the data node, and which scenarios can access it.
```
since we are using access in "which scenarios can access it." Would it be more relevant to use the term "accessibility"?